### PR TITLE
feat(SHP-1513): inventory items

### PR DESCRIPTION
**Pull request by ezelopes-mohc**
Inventory items repository and struct.

### CODE REVIEW

File: inventory-items.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+// InventoryLevel represents the amount of an item that there is available in a shop
+type InventoryItem struct {
+	// The ID of the inventory item
+	ID int64
+	// The unique SKU (stock keeping unit) of the inventory item
+	SKU string
+	// The date and time when the inventory level was created
+	CreatedAt string
+	// The date and time when the inventory level was last modified
+	UpdatedAt string
```
Comment (rob-mohc) [inline inventory-items.go:23]: Shouldn't this `CreateAt` and `UpdatedAt` be of type `Time`?

File: inventory-items.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	// Whether a customer needs to provide a shipping address when placing an order containing the inventory item
+	RequiresShipping bool
+	// The unit cost of the inventory item
+	Cost string
+	// The country code of where the item came from (ISO)
+	CountryCodeOfOrigin string
+	// The province code of where the item came from (ISO)
+	ProvinceCodeOfOrigin string
+	// The general Harmonized System (HS) code for the inventory item. Used if a country-specific HS code (`countryHarmonizedSystemCode`) is not available.
+	HarmonizedSystemCode int
```
Comment (rob-mohc) [inline inventory-items.go:33]:
```suggestion
	HarmonizedSystemCode int64
```
Should this be int64?

File: inventory-items.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package shopify
+
+// InventoryItemRepository manages inventory items
+type InventoryItemRepository interface {
+	// Get retrieves an inventory item by its id
+	Get(id int64) (InventoryItem, error)
+}
+
+type CountryHarmonizedSystemCode struct {
```
Comment (rob-mohc) [inline inventory-items.go:9]: Think `CountryHarmonizedSystemCode` should have some description since is an exported type

**Review by rob-mohc — CHANGES_REQUESTED** [review]
Looks good, added some small comments

File: inventory-items.go (hunk 3 above)
Comment (ezelopes-mohc) [reply inventory-items.go:9]: Ah it doesn't need to be exported, good shout 🙂

**Review by rob-mohc — APPROVED** [review]

**Pull request merged**